packages/nodes-base/nodes/Slack/V2/GenericFunctions.ts

```diff
@@ -162,7 +162,12 @@ export function getMessageContent(
 			};
 			break;
 		case 'block':
-			content = jsonParse(this.getNodeParameter('blocksUi', i) as string);
+			const blocksUi = this.getNodeParameter('blocksUi', i) as string | IDataObject;
```

Comment:
Since the v2 node defines this property as `string`, and v1 node does not call this code, if `blocksUi` is an object under any circumstances, then we have a much deeper problem elsewhere in code. 
This change might fix the issue for this specific case, but it's not a great solution, since issues like these could pop up elsewhere.

Reply:
@netroy 

in v1 we are using json as type
![image](https://github.com/n8n-io/n8n/assets/88898367/7c56a7fa-4302-41f3-9c85-75ae9177d893)
in v2 string
![image](https://github.com/n8n-io/n8n/assets/88898367/b119318a-87ae-4c78-8926-a948f167b24b)

I'm not aware why this change was made, and if it was intentional